plugin/trino-pinot/src/main/java/io/trino/plugin/pinot/PinotColumnHandle.java

```diff
@@ -44,7 +41,7 @@
 
     public PinotColumnHandle(String columnName, Type dataType)
     {
-        this(columnName, dataType, columnName, false, false, true, Optional.empty(), Optional.empty());
+        this(columnName, dataType, quoteIdentifier(columnName), false, false, true, Optional.empty(), Optional.empty());
```

Comment:
It would be better to store an original column name (not quoted) in `PinotColumnHandle` and quote it when needed. Could you leave TODO comment since it's pre-existing issue?

Reply:
I updated to quote when needed. Do you still recommend a TODO comment?
(also saved previous version just in case)